pkg/virt-handler/vm_test.go

```diff
@@ -241,9 +241,12 @@ var _ = Describe("VirtualMachineInstance", func() {
 	AfterEach(func() {
 		close(stop)
 		wg.Wait()
-
+		events := []string{}
+		for len(recorder.Events) > 0 {
+			events = append(events, <-recorder.Events)
+		}
```

Comment:
Won't this break what is described in the comment below? You're removing the need to explicitly check events?